packages/discovery/src/discovery/provider/TransactionTrace.ts

```diff
@@ -0,0 +1,33 @@
+import * as z from 'zod'
+
+const TraceAction = z.object({
+  callType: z.string().optional(),
+  from: z.string(),
+  to: z.string().optional(),
+  value: z.string(),
+  input: z.string(),
+  gas: z.string(),
+  init: z.string().optional(),
+})
+
+const TraceResult = z.object({
```

Comment:
If we really don't want then we don't have to, but if it isn't a problem we can expose this type

Reply:
I prefer to export only things that have potential to be used somewhere and I don't frankly see it here.